packages/home-service/src/resolver.ts

```diff
@@ -1,6 +1,7 @@
-import { Home } from './schema';import { pubsub } from './helpers';
+import { Home } from './schema';
+import { pubsub } from './helpers';
 
-const Home_UPDATE = "Home_UPDATE";
+const Home_UPDATE = 'Home_UPDATE';
 
 export const HomeResolver = {
   Subscription: {
```

Comment:
SUbscriptions don't work with federations directly. Please test your code

Reply:
That's not been worked upon in this PR, this is just the basic structure that has been generated by the CLI with types.

Reply:
@riginoommen I have removed the subscription as home-spa is not going to need it for now